mobile/test/kotlin/integration/EngineApiTest.kt

```diff
@@ -8,36 +8,29 @@ import io.envoyproxy.envoymobile.engine.JniLibrary
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.After
 import org.junit.Test
 
 class EngineApiTest {
-  private var engine: Engine? = null
-
   init {
     JniLibrary.loadTestLibrary()
   }
 
-  @After
-  fun teardown() {
-    engine?.terminate()
-    engine = null
-  }
-
   @Test
   fun `verify engine APIs`() {
     val countDownLatch = CountDownLatch(1)
-    engine = EngineBuilder()
+    val engine = EngineBuilder()
```

Comment:
I am not even sure whether this tests does anything useful since we do not call addGrpcDomain method on the engine so we are effectively not even enabling stats sink... will probably just disable it if it keeps being flaky.